src/main/java/org/hawkular/client/android/backend/model/Alert.java

```diff
@@ -27,7 +29,8 @@
 import android.support.annotation.VisibleForTesting;
 
 public final class Alert implements Parcelable {
-    @SerializedName("alertId")
+    @RecordId
```

Comment:
Yeah. Faced a lot of difficulty cause progourd was wiping the method and program was not working without it.
Its how it is made to be.
Alert ACK is now with a PUT request